Makefile

```diff
@@ -723,6 +724,10 @@ out/39-ohio/state.gpkg: data/39-ohio/statewide/2010/tl_2012_39_vtd10.zip \
 	ogr2ogr -sql "SELECT '2016' AS year, '39' AS state, '021' AS county, CONCAT('39021', CAST(District_N AS character(10))) AS precinct, 'polygon' AS accuracy FROM CHP_PREC" \
 		-s_srs '+proj=lcc +lat_1=38.73333333333333 +lat_2=40.03333333333333 +lat_0=38 +lon_0=-82.5 +x_0=600000 +y_0=0 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=us-ft +no_defs' \
 		-t_srs EPSG:4326 -nln state -append -f GPKG $@ 'out/39-ohio/source/CHP_PREC.shp'
+	unzip -d out/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.zip
```

Comment:
This one tossed me for a loop:

> unzip -d out/39-ohio/source data/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.zip

versus:

> unzip -d out/39-ohio/39061-hamilton/HamiltonCountyOHPrecincts_2016.zip

The output and input both need to be specified else unzip will choke.

Reply:
Aha, good catch. Messed up a paste on that. Fixed.